wp-modules/helper-functions/helper-functions.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Module Name: Helper Functions
+ * Description: This module contains helper functions which can be used by other modules.
+ * Namespace: HelperFunctions
+ *
+ * @package pattern-manager
+ */
+
+declare(strict_types=1);
+
+namespace PatternManager\HelperFunctions;
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Run a string of HTML through the_content filters. This makes it so everything needed will be rendered in wp_footer.
+ *
+ * @param string $content The html content to run through the filters.
+ * @return bool
+ */
+function do_the_content_things( $content ) {
```

Comment:
Does this need its own module, given that it's only called in `wp-modules/pattern-preview-renderer/pattern-preview-renderer.php` ?

Reply:
I moved it out because do_the_content_things was being used in 2 different modules, but since https://github.com/studiopress/pattern-manager/pull/148 I guess that's no longer the case. 

I don't mind having some of these "module agnostic" functions in a dedicated module. It might perhaps make sense as part of @mike-day's work in https://github.com/studiopress/pattern-manager/pull/141 though. Mabe we leave it for now and look at moving it into his module once that's ready?

Reply:
Good idea!